genesis/embedded.go

```diff
@@ -0,0 +1,25 @@
+package genesis
+
+import (
+	"time"
+
+	"github.com/OffchainLabs/prysm/v6/beacon-chain/state"
+	"github.com/OffchainLabs/prysm/v6/config/params"
+	"github.com/OffchainLabs/prysm/v6/genesis/embedded"
+)
+
+var embeddedGenesisData map[string]GenesisData
+
+func init() {
+	embeddedGenesisData = make(map[string]GenesisData)
+	embeddedGenesisData[params.MainnetName] = GenesisData{
+		ValidatorsRoot: [32]byte{75, 54, 61, 185, 78, 40, 97, 32, 215, 110, 185, 5, 52, 15, 221, 78, 84, 191, 233, 240, 107, 243, 63, 246, 207, 90, 210, 127, 81, 27, 254, 149},
+		Time:           time.Unix(1606824023, 0),
+		embeddedBytes: func() ([]byte, error) {
+			return embedded.BytesByName(params.MainnetName)
+		},
+		embeddedState: func() (state.BeaconState, error) {
+			return embedded.ByName(params.MainnetName)
+		},
+	}
```

Comment:
Can you add a test to show that these hardcoded values match the computed values from the embedded mainnet.ssz?

Reply:
Resolved in 66f3e399581b49affa31de3e18d9b503ae6e7e8e